controllers/login.js

```diff
@@ -99,8 +248,9 @@ const determineRedirectUrl = (req) => {
 	return '/dashboard';
 };
 
-const getNonOauthSchools = (schools) => [...schools]
-	.filter((school) => school.systems.filter((system) => system.oauthConfig || system.type === 'oidc').length === 0);
+const filterSchoolsWithLdapLogin = (schools) => schools
+	// eslint-disable-next-line max-len
+	.filter((school) => school.systems.some((system) => system.type === 'ldap' && !system.oauthConfig));
```

Comment:
oidcConfig would not be excluded - not sure if you want it or not
fyi, the backend api/v1/schoolsList returns '_id type alias oauthConfig.provider'

Reply:
The reason why we filter for oauthConfig is that Iserv has the type 'ldap' for the ldap sync, but the login works with oauth2 and we had the requirement to remove all Iserv systems from the ldap login list. I don't know if this is a use case for oidc too, since the old filter function here did not filter for oidcConfig, but for the type.